python/mxnet/numpy/multiarray.py

```diff
@@ -282,6 +313,16 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):  # pylint: disable=
                     _FALLBACK_ARRAY_UFUNC_WARNED_RECORD[onp_op] = True
                 out = onp_op(*new_inputs, **kwargs)
                 return _as_mx_np_array(out, ctx=inputs[0].ctx)
+            # ops with np mx_np
+            elif name in ufunc_list and isinstance(inputs[0], _np.ndarray):
+                # inplace
+                if 'out' in kwargs:
+                    new_inputs = [arg.asnumpy() if isinstance(arg, ndarray) else arg for arg in inputs]
+                    return onp_op(*new_inputs, **kwargs)
+                else:
+                    new_inputs = [_as_mx_np_array(arg, ctx=inputs[1].ctx)
```

Comment:
I'm not sure why should we use `inputs[1].ctx` instead of `inputs[0].ctx`.

Reply:
Ignore my previous comment. I mean to choose one element in the inputs that has a valid `ctx`.